tests/benchmark_test_inputs/Economic_KPI_C2_E2/csv_elements/storage_1.csv

```diff
@@ -11,4 +11,6 @@ dispatch_price,currency/kWh,NA,0,0
 soc_initial,None or factor,None,NA,NA
 soc_max,factor,1,NA,NA
 soc_min,factor,0.2,NA,NA
+fixed_thermal_losses_relative,factor,0,,
+fixed_thermal_losses_absolute,kWh,0,,
```

Comment:
We used `NaN` ("NA") for parameters that do not apply to one of the sub-assets. It would be better if you use that here, too, instead of empty cells.
Same for the other `storage_*.csv` input fiels. 

Is there a specific reason that you included the parameters in the benchmark test of `Economic_KPI_C2_E2`? I would have thought that it is only stored in the one benchmark test that checks if the feature works for you).

Reply:
> Is there a specific reason that you included the parameters in the benchmark test of `Economic_KPI_C2_E2`? I would have thought that it is only stored in the one benchmark test that checks if the feature works for you).

Sorry this was still in from before. Deleted with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/0e0f60012df350bb773d3135bd8b4a5d744268f6

Reply:
> Same for the other `storage_*.csv` input fiels.

Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/d0e941965535f085a0d1c30cbae2614d77093a75